test/core.js

```diff
@@ -60,12 +60,48 @@ QUnit.test( "jQuery( '#' )", function( assert ) {
 	assert.expect( 2 );
 
 	expectWarning( assert, "Selector, through the jQuery constructor, nothing but hash",
-			function() {
```

Comment:
There are some spurious diffs here from line lengths exceeding 100 chars that I had to fix.